tools/obisindicators/obisindicators.xml

```diff
@@ -0,0 +1,128 @@
+<tool id="obisindicators" name="Ocean biodiversity indicators" version="@VERSION@" profile="20.01">
+    <description>from OBIS</description>
+    <macros>
+        <import>macro.xml</import>
+    </macros>
+    <expand macro="topic"/>
+    <expand macro="requirements">
```

Comment:
One things that just popped to my mind is that it would be nice to annotate all needed files for this tool (so the R scripts) with https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-required-files-include ...

This will help in the pulsar context, where the scripts also need to be transferred.